packages/core/src/kilt/Kilt.ts

```diff
@@ -35,13 +36,23 @@ export async function init<K extends Partial<ConfigService.configOpts>>(
  * Connects to the KILT Blockchain and passes the initialized api instance to `init()`, making it available for functions in the sdk.
  *
  * @param blockchainRpcWsUrl WebSocket URL of the RPC endpoint exposed by a node that is part of the Kilt blockchain network you wish to connect to.
+ * @param apiOpts Additional parameters to be passed to ApiPromise.create().
+ * @param apiOpts.noInitWarn Allows suppressing warnings related to runtime types and augmentation. Enabled by default if the global log level is 'error' or higher.
```

Comment:
By synchronizing the levels I meant that these *warnings* should show when the log level in the SDK is set to *warn*. But I don’t really mind the current implementation.

Reply:
that's what's happening - error level 3 is warn, anything above disables it.

Reply:
Thank you for the improved documentation!